torch_xla/csrc/xla_backend_impl.cpp

```diff
@@ -125,7 +125,8 @@ class XlaBackendImpl : public torch::lazy::BackendImplInterface {
       // TODO(JackCaoG): Verify this with GPU, we might only have 1 process with
       // multiple GPU as replica.
       compile_instances.push_back(xla::ComputationClient::CompileInstance(
-          torch_xla_computation->move_computation(), current_device.toString(),
+          torch_xla_computation->move_computation(),
+          torch_xla_computation->get_device_string(),
```

Comment:
shoudn't this be,
```
(
  torch_xla_computation->move_computation(), 
  current_device.toString(),
  {torch_xla_computation->get_device_string()}, &shape
)
```
since there is one compilation device, and we can remove the 1-device assumption in `get_device_string()`.

Reply:
cc @JackCaoG